### DISCUSSION TITLE
Clarification on scope of CRWG and DPBE

### DISCUSSION THREAD

**Discussion by chrisisla** [finos/common-domain-model#3254] [Meetings]
The discussion at the meeting on the CRWG and DPBE raised a few potential changes to the CDM working group structure.

Firstly, I believe it was clear that the CRWG should remain as a cross domain working group that is used as a forum for work that has already been done to be approved for inclusion in the CDM. 

Secondly, the conversation raised the need for a forum where questions pertaining to modelling and scope of changes could be discussed prior to work being done on specific issues. At the moment these discussions are – and will continue to be – held in domain specific working groups. However, the majority of the issues raised affect multiple domains, thus the need for a generic modelling meeting. 

I propose the following changes for review by the community:
1.	Keep the CRWG as a cross domain working group reviewing post-work changes
2.	Change the DPBE to be a cross-domain modelling workgroup reviewing pre-work changes
3.	To reflect importance of the CRWG it should be permanently chaired by a TA maintainer
4.	The updated DPBE could be chaired on a rotating basis (as per the CRWG is currently) or by community members

Please add your thoughts and comments so we can decide on what (if any) changes need to be considered.

**Comment by lolabeis** [comment]
Closing as ratified by the [SWG](#3308)
